api/v1alpha08/kogitoserverlessworkflow_types.go

```diff
@@ -35,8 +56,9 @@ type KogitoServerlessWorkflowStatus struct {
 	// +optional
 	Address duckv1.Addressable `json:"address,omitempty"`
 	// keeps track of how many failure recovers a given workflow had so far
-	RecoverFailureAttempts int       `json:"recoverFailureAttempts,omitempty"`
-	Endpoint               *apis.URL `json:"endpoint,omitempty"`
+	RecoverFailureAttempts int         `json:"recoverFailureAttempts,omitempty"`
+	LastTimeRecoverAttempt metav1.Time `json:"lastTimeRecoverAttempt,omitempty"`
```

Comment:
How can be useful the information in LastTimeRecoverAttempt ?

Reply:
Please take a look at the devmode reconciliation algorithm in this PR. I even wrote a comment about this.